torch/_dynamo/variables/torch.py

```diff
@@ -676,11 +676,21 @@ def fn_with_prim_types(x):
                 and self.value in tensortype_to_dtype
                 and len(args) == 1
                 and isinstance(args[0], ListVariable)
-                and args[0].is_python_constant()
             ):
-                for x in args[0].items:
-                    if isinstance(x.value, np.generic):
-                        x.value = x.value.item()
+                # torch.LongTensor cannot accept a list of FakeTensors.
+                # So we stack the list of FakeTensors instead.
+                if all(isinstance(x, variables.TensorVariable) for x in args[0].items):
```

Comment:
you can now lift this to the previous if. Add also `and len(args[0].items) > 1`

Reply:
done